src/app.js

```diff
@@ -2,26 +2,31 @@ import express from "express";
 import dotenv from "dotenv";
 import morgan from "morgan";
 import { sequelize } from "./database/models";
-import router from "./routes/router";
+import register from "./routes/router.js";
 
 dotenv.config();
 
 const app = express();
 
 export const connectDB = async () => {
-    try {
-        await sequelize.authenticate();
-        console.log("Database connection established successfully");
-    } catch (err) {
-        console.log(`Database connection failed: ${err}`);
-        process.exit(1);
-    }
+  try {
+    await sequelize.authenticate();
+    console.log("Database connection established successfully");
+  } catch (err) {
+    console.log(`Database connection failed: ${err}`);
+    process.exit(1);
+  }
 };
 
 app.use(morgan("dev"));
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
+app.use("/", register);
```

Comment:
This line is not allowed to be here, because below this line there is line connecting you to routes

Reply:
Can you explain a bit more on that?